lib/import_export/query.rb

```diff
@@ -21,7 +21,7 @@ def endpoint
       fuzzy_name: false,
       type: nil,
       types: nil,
-      size: 100,
+      size: 50,
```

Comment:
Might be good to raise an argument error if it's over 50 to keep things less surprising.